books/RayTracingTheRestOfYourLife.html

```diff
@@ -298,7 +298,7 @@
 
         cam.defocus_angle = 0;
```

Comment:
Not sure if this _really_ matters, but in the source code `cam.defocus_angle` is `0.0` (rather than just `0`)

same with `vfov` (40.0 instead of 40). 

I can make a separate issue, or you an quickly address them here.

Reply:
I'll just do a quick fix after this.